types/expo/tslint.json

```diff
@@ -1 +1,8 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "strict-export-declare-modifiers": false,
```

Comment:
Can you re-enable these please?

Reply:
@DanielRosenwasser, yeah, sure. I have done.